src/adapters/ActivitiesJSONAdapter.test.js

```diff
@@ -248,21 +248,33 @@ describe('Activities JSON Adapter', () => {
             type: 'ColumnSet',
             columns: [
               {
-                type: 'Input.Date',
-                id: 'arrivalDate',
-                height: 'auto',
-                isRequired: true,
-                errorMessage: 'Arrival Date is required',
-                label: 'Arrival Date',
+                type: 'Column',
+                width: 'stretch',
```

Comment:
width: 1

'stretch' should be used when the rest of the siblings have 'auto' width